app/Http/Controllers/CompetitionsController.php

```diff
@@ -206,4 +209,40 @@ public function leaderboard(Competition $competition)
 
         return view('competitions.leaderboard', compact('competition', 'leaderboard', 'pending', 'flagged'));
     }
+
+    /**
+     * Get the featured reportback form.
+     *
+     * @param  \Gladiator\Models\Competition  $competition
+     * @return \Illuminate\Http\Response
+     */
+    public function featuredReportbackForm(Competition $competition, Message $message)
```

Comment:
should this be `editFeaturedReportback`... we use the `index, edit, show, update, delete` language in controllers and you do it below w/ `updateFeaturedReportback`